instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
   compileOnly("org.apache.logging.log4j:log4j-core:2.17.0")
   implementation(project(":instrumentation:logback:logback-appender-1.0:library"))
   compileOnly("ch.qos.logback:logback-classic:1.0.0")
+  implementation(project(":instrumentation:jdbc:library"))
```

Comment:
Would all Spring Boot autoconfigure projects use JDBC, or does there need to be a new autoconfigure module that includes JDBC?

Reply:
Not sure to follow.

The JDBC API is part of the JDK. 

For the Logback instrumentation, at line 30 we can see that `instrumentation:logback:logback-appender-1.0:library` is included but the `ch.qos.logback:logback-classic:1.0.0` library is `compileOnly` (the Logback telemetry is only enabled when the user adds this library).

@kenfinnigan I hope I have answered your question

Reply:
Apologies, I asked my question badly.

Though the JDBC APIs are part of the JDK, are there any drawbacks to including the instrumentation into all Spring Boot projects using the autoconfigure module? Even if they don't use JDBC